functions/firestore/users/onDelete.f.js

```diff
@@ -0,0 +1,13 @@
+import { firestore } from 'firebase-functions';
+import admin from 'firebase-admin';
+
+export default firestore
+  .document('users/{userId}')
+  .onDelete((snap, context) => {
+    const { userId } = context.params;
+    const bucket = admin.storage().bucket();
+
+    return bucket.deleteFiles({
+      prefix: `users/${userId}`,
```

Comment:
Do you mean to delete the user from Firestore instead of from Storage?

Reply:
Oh whoops, I meant to delete the user from Oauth if it's been deleted from Firestore, although I'm not sure if this is necessary.